nettests/websites/web_connectivity.go

```diff
@@ -1,10 +1,56 @@
 package websites
 
 import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
 	"github.com/measurement-kit/go-measurement-kit"
 	"github.com/openobservatory/gooni/nettests"
+	"github.com/pkg/errors"
 )
 
+type URLInfo struct {
+	URL          string `json:"url"`
+	CategoryCode string `json:"category_code"`
+}
+
+type URLResponse struct {
+	Results []URLInfo `json:"results"`
+}
+
+const orchestrateBaseURL = "https://events.proteus.test.ooni.io"
+
+func lookupURLs(ctl *nettests.Controller) ([]string, error) {
+	var (
+		parsed = new(URLResponse)
+		urls   []string
+	)
+	reqURL := fmt.Sprintf("%s/api/v1/urls?probe_cc=%s",
```

Comment:
add a Todo to include other config constraints here, like which categories the user is okay with,
I guess also - in the orchestrator url request, does the client include any form of ID of itself to allow linking of the different requests its making to proteus?

Reply:
Yeah for sure. The nettest class is designed to have knowledge of the config options via `ctl.Config.Foo`, but currently config support is not complete. As soon as we have the chosen categories as part of the config, it will be possible to pass those to the orchestrator.